pyproj/geod.py

```diff
@@ -1029,7 +1029,10 @@ def geometry_area_perimeter(
         the 2 loops in a figure-8 polygon will partially cancel.  There's no need
         to "close" the polygon by repeating the first vertex.
 
-        .. note:: lats should be in the range [-90 deg, 90 deg].
+        .. note:: | There are a few limitations :
+                  | - lats should be in the range [-90 deg, 90 deg] ;
```

Comment:
Is that a limitation? There could be discussions about the longitudes, but latitudes above 90 deg does not make sense so me.